pkg/attest/platform_cert_fetcher.go

```diff
@@ -166,33 +173,36 @@ func fetchWithRetry(requestURL string, baseSec int, maxRetries int) ([]byte, err
 		}
 		res, err := http.Get(requestURL)
 		if err != nil {
+			logrus.Debugf("fetch on retry %d: http.Get failed: %s", retryCount, err)
 			retryCount++
 			continue
 		}
 		if 200 <= res.StatusCode && res.StatusCode < 300 {

```

Comment:
What's the point of the `&& < 300`?  The first check also enforces the second check.

Reply:
The `<=` really through me off (laugh)